classes/edge.bbclass

```diff
@@ -0,0 +1,89 @@
+#class for Pelion Edge varibles and functionality
```

Comment:
Quite a few typoes in this file, I'd recommend going through it

Reply:
Thanks, update coming